tuning_node:
  ros__parameters:
    est_twist_topic: "/pronto_estimator/twist"
    est_pose_topic: "/pronto_estimator/pose"
    odom_twist_cor: "/pronto_estimator/Odom_Correction"
    folder_path: "/home/jacopo/Desktop/Pronto_Tuning/"

command_node:
  ros__parameters:
    topic_name: "motion_generator/simple_velocity_command"
    set_point_list: 
      - start
      - move_for
      - straight
      - slow_down
      - stop 
    start: 
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 0.0
    move_for: 
      vx: 0.15
      vy: 0.0
      omega: 0.0
      time: 1.0
    straight: 
      vx: 0.15
      vy: 0.0
      omega: 0.0
      time: 51.0
    slow_down: 
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 52.0
    stop: 
      vx: 0.0
      vy: 0.0
      omega: 0.0
      time: 55.0
    use_sim_time: true